package-lock.json

```diff
@@ -0,0 +1,3762 @@
+{
```

Comment:
We don't usually commit these lockfiles. Could it be added to the .gitignore?

Reply:
Apparently, it already is in the `.gitignore` file. I've dropped it from this PR.